docs/api-reference/google-maps/overview.md

```diff
@@ -29,6 +29,16 @@ npm install @deck.gl/core @deck.gl/google-maps
 import {GoogleMapsOverlay} from '@deck.gl/google-maps';
 ```
 
+## Vector/Raster maps
+
+Starting with Google Maps v3.45 there are two modes of rendering [Vector and Raster](https://developers.google.com/maps/documentation/javascript/vector-map). To control which rendering mode is used, you need to configure the Google Map using the [Google Cloud Platform](https://developers.google.com/maps/documentation/javascript/webgl).
+
+From v8.6, the @deck.gl/google-maps overlay automatically detects at runtime which rendering type is used. The Vector rendering mode is in general more performant, and the deck.gl overlay offers several features not available when using Raster rendering:
```

Comment:
Different terms are used interchangeably in this section ("@deck.gl/google-maps overlay", "deck.gl overlay") and may cause some confusion. I suggest using:

- **`@deck.gl/google-maps` module** to refer to this package
- **`GoogleMapsOverlay` class** to refer to the Google Maps OverlayView implementation 
- **deck.gl** to refer to the visualization framework/rendering engine